.woodpecker/main.yml

```diff
@@ -59,13 +61,28 @@ pipeline:
       branch: ${CI_REPO_DEFAULT_BRANCH}
       event: push
 
+  publish-agent-dryrun:
+    group: bundle
+    image: woodpeckerci/plugin-docker-buildx
+    privileged: true
+    settings:
+      dry_run: true
+      repo: woodpeckerci/woodpecker-agent
+      dockerfile: docker/Dockerfile.agent.multiarch
+      platforms: linux/arm/v7,linux/arm64/v8,linux/amd64,linux/ppc64le,windows/amd64,darwin/amd64,darwin/arm64
+      tag: test
+    when:
+      event: pull_request
+
   publish-agent:
     group: bundle
-    image: plugins/docker
+    image: woodpeckerci/plugin-docker-buildx
+    privileged: true
```

Comment:
if https://github.com/woodpecker-ci/woodpecker/pull/623 got merged we can remove it